lib/oelite/fetch/git.py

```diff
@@ -267,6 +267,17 @@ def unpack(self, d):
         return True
 
     def mirror(self, mirror=os.getcwd()):
+        if self.is_local:
+            # Don't mirror if the source is local. Concept taken from fetch
+            # Do check revision exists before accepting though
+            print "Warning: Skipping creating mirror for local path %s"%(self.uri)
```

Comment:
I agree, it is a good thing to notify the user that stuff is not being mirrored.  But it should generally be possible to avoid warnings, which mandates that this notification is not really warning.